index.html

```diff
@@ -6281,7 +6336,45 @@ <h3><dfn>Execute Async Script</dfn></h3>
       and data <var>result</var>.
 </ol>
 </section> <!-- /Execute Async Script -->
-</section> <!-- /Executing Script -->
+<section>
+ <h3><dfn>Pin Script</dfn></h3>
+
+ <table class="simple jsoncommand">
+  <tr>
+   <th>HTTP Method</th>
+   <th>URI Template</th>
+  </tr>
+  <tr>
+   <td>POST</td>
+   <td>/session/{<var>session id</var>}/execute/pin</td>
+  </tr>
+ </table>
+
+ <p>The <a>remote end steps</a> are:
+
+  <ol>
+   <li><p>Let <var>name</var> be the result of
+ <a>getting a property</a> named <var>name</var>
+ from the <var>parameters</var>.
+
+ <li><p>Let <var>script</var> be the result of
+ <a>getting a property</a> named <var>script</var>
+ from the <var>parameters</var>.
+
+ <li><p>If <var>script</var> is <a>undefined</a> or is not a <a>String</a>,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>If <var>name</var> is <a>undefined</a> or is not a <a>String</a>,
+ return <a>error</a> with <a>error code</a> <a>invalid argument</a>.
+
+ <li><p>If <var>name</var> is equal to a key in the <a>executable script map</a>
+ remove the <a>executable script</a> from the session
+
+ <li><p>add the <a>executable script</a> to the session.
```

Comment:
To be more in line with how [Infra suggests using maps](https://infra.spec.whatwg.org/#maps), both of these steps can be replaced with:

```
<li><p>Set <a>executable script map</a>[<var>name</var>] to <var>script</var>.
```